multimedia/minisatip/files/minisatip.init

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=80
+
+APP=minisatip
+
+start() {
+        service_start /usr/bin/$APP -R /usr/share/minisatip/html
+}
+
+stop() {
+        service_stop /usr/bin/$APP
+}
+
```

Comment:
Please use procd here.  You can find a guide on this at the wiki https://openwrt.org/docs/guide-developer/procd-init-scripts .  I often find it's simpler just take an existing one as example: https://github.com/yousong/waller/blob/master/utdnsd/files/utdnsd.init